apps/storefront/pnpm-lock.yaml

```diff
@@ -0,0 +1,16974 @@
+lockfileVersion: 5.3
```

Comment:
apps/storefront does no longer exist in develop so maybe a rebase gone wrong? Please remove this file :)